packages/api/models/rules.js

```diff
@@ -35,6 +35,12 @@ class Rule extends Manager {
     return r.RuleArn;
   }
 
+  async deleteRules() {
+    const rules = await this.scan();
+    const deletePromises = rules.Items.map((r) => this.delete(r));
+    await Promise.all(deletePromises);
```

Comment:
Nit: `async` functions always return a promise. So if the final action in an `async` function is to `await` a promise, it's equally effective to just return the promise, because `await deleteRules` will await the returned promise